c2cgeoportal/views/entry.py

```diff
@@ -1276,11 +1277,12 @@ def logout(self):
         return set_common_headers(
             self.request, "logout", NO_CACHE,
             response=Response("true", headers=headers),
+            add_cors=True
         )
 
     @view_config(route_name="loginchange", renderer="json")
     def loginchange(self):
-        set_common_headers(self.request, "loginchange", NO_CACHE)
+        set_common_headers(self.request, "loginchange", NO_CACHE, add_cors=True)
```

Comment:
Maybe CSRF is not the correct term to use. I am not sure. BUT I do think enabling CORS creates a breach here. This makes it possible to create a page that can change the user's password by sending an Ajax request to GeoMapFish's loginchange web service. No?